lib/open_food_network/products_cache_refreshment.rb

```diff
@@ -21,21 +21,23 @@ def self.refresh(distributor, order_cycle)
       enqueue_job(job) unless pending_job?(job)
     end
 
-    private
-
-    def self.refresh_job(distributor, order_cycle)
-      RefreshProductsCacheJob.new(distributor.id, order_cycle.id)
-    end
-
-    def self.pending_job?(job)
-      Delayed::Job.
-        where(locked_at: nil).
-        where(handler: job.to_yaml).
-        exists?
-    end
-
-    def self.enqueue_job(job)
-      Delayed::Job.enqueue job, priority: 10
+    class << self
```

Comment:
what happened here?
You sent me reading about eigenclasses... my head hurts :-)
In this particular case, is this a convention (use class << self instead of self. ) or is there any change in logic with this?

Reply:
I'd like to check what the Ruby Style guide says about it and I'd stick to the community's default. Also, `private` doesn't work for class methods AFAIK, so I'm not sure here if this methods get hid.

Reply:
hmm.. I read a bit more about this and actually, this is exactly how you create private class methods :-)

Reply:
I know! I was a bit reluctant to use `class << self` here, but I followed rubocop and found that it is the widely accepted way to declare private class methods. I should have tackled one rubocop violation at a time to explain it better. Sorry.

The alternative is to declare it separately:
```ruby
private_class_method :refresh_job
private_class_method :pending_job?
private_class_method :enqueue_job
```

That is more repetitive, but you definitely know what it's doing. Should I change it?

Reply:
:+1:
private_class_method is much easier to understand :-)
If it's also correct I'd use it.

Reply:
yes, imo, much better.